wpl_cli/README.md

```diff
@@ -0,0 +1,114 @@
+# Wen Program Library CLI
+
+A command line interface for WEN New Standard program.
+
+---
+
+## Table of Contents
+
+- [Installation](#installation)
+- [Instructions](#instructions)
+
+---
+
+## Installation
+
+One can build directly from source using `cargo`. Prerequisites for the binary to work is installing Rust on one's machine.
+
+Install Rust via the command
+
+```sh
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+```
+
+---
+
+## Instructions
```

Comment:
Instead of instructions can we you please swap with an example section. 

- Creating a group
- Creating an asset
- Adding an asset to a group